src/data/site-nav-structure.json

```diff
@@ -2,7 +2,7 @@
   "getting-started": {
     "title": "Getting Started",
     "subnav": {
-      "carbon-and-duo": "Carbon and Duo",
+      "carbon-and-duo": "IBM Design Language",
```

Comment:
Should we change the subnav title, as well? The `"carbon-and-duo"` bit.

Reply:
That's the actual url, we could probably add a redirect though until https://w3.ibm.com/design/essentials  can update their link.

Reply:
If we want to do a redirect on our side then yes, we can just resolved everything and update our subnav to "design-language" or something similar.

Reply:
The title of the page is generated from the nav (I think) so we need to update everything all at once. Not sure on how to setup redirects with this version of the website so we should coordinate with the w3.ibm.com site on updating their url.

Reply:
@shinytoyrobots can go ahead and update the title and nav link. Then we can submit a PR here to update the link on the design website https://github.ibm.com/Design/w3-design-site/blob/master/pages/design/essentials.js#L182